akka-cluster/src/main/scala/akka/cluster/ClusterDaemon.scala

```diff
@@ -917,6 +915,7 @@ private[cluster] class ClusterCoreDaemon(publisher: ActorRef) extends Actor with
    */
   def gossipRandomN(n: Int): Unit = {
     if (!isSingletonCluster && n > 0) {
+      // TODO what about local vs cross dc here?
```

Comment:
It's a best effort when shutting down. It could be good to do some of both, e.g. when stopping last node in a dc it can be good that it's spread to other dc.
1 to other dc, and the rest to own dc?